src/app/content/highlights/components/Confirmation.tsx

```diff
@@ -53,7 +51,7 @@ const Confirmation = React.forwardRef<HTMLElement, Props>((
   ref
 ) => {
   return <Overlay
-    ref={mergeRefs(ref, useDrawFocus())}
+    ref={ref}
```

Comment:
maybe we could test this, but i think when deleting a highlight we still want this behavior. maybe you could add a drawFocus prop to the Confirmation component and do
```suggestion
    ref={drawFocus ? mergeRefs(ref, useDrawFocus()) : ref}
```

if the keyboard tab order naturally falls into the confirmation window when deleting a highlight, then it doesn't matter and we can leave it as you have it here

Reply:
In testing, the tab order did fall into order, but without the focus call, the menu overlapped it until you hit tab, so I think the change you proposed is a good one, updated the PR